internal/providers/pluginfw/resources/user/data_users_acc_test.go

```diff
@@ -0,0 +1,62 @@
+package user_test
+
+import (
+	"testing"
+
+	"github.com/databricks/terraform-provider-databricks/internal/acceptance"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const dataSourceTemplate = `
+	resource "databricks_user" "user1" {
+		user_name = "testuser1@databricks.com"
+	}
+
+	resource "databricks_user" "user2" {
+		user_name = "testuser2@databricks.com"
+	}
+
+	data "databricks_users" "this" {
```

Comment:
This needs a `depends_on` field to wait for the users to be created before listing.

Reply:
Done 👍